.perltidyrc

```diff
@@ -0,0 +1,18 @@
+# Workaround needed for handling non-ASCII in files.
```

Comment:
See https://github.com/os-autoinst/os-autoinst/blob/master/xt/00-tidy.t vs. https://github.com/os-autoinst/openQA/blob/master/.perltidyrc

So we would have to decide about which config we want to use in the future.

Reply:
Looks like `tidyall_ok` uses the default config file:

https://github.com/houseabsolute/perl-code-tidyall/blob/10425fb8e720c06f96d3935bc98b2955fc3e3294/lib/Test/Code/TidyAll.pm#L23-L29

looks like it'd be a matter of providing this one as a symlink and that would be it ?

Reply:
Yeah, the point is, if we use the symlink to this one here, we would first need to decide which of the two different configs we want.
I think it's better if we do that first by providing single PRs to both repos with the suggested new config and look at the tidy changes, so we can decide.